### ISSUE TITLE
SystemLoad Agent Plugin - Typo

### ISSUE THREAD

**Issue by kbilev** [it-novum/openITCOCKPIT#1600]
**Describe the bug**
In the file "/opt/openitc/receiver/src/itnovum/openITCOCKPIT/Checks/Receiver/Plugins/SystemLoad.php" is a typo.
In the timerange switch, there are two cases for "5", but none for 15.
```
case 5:
                  // use load15
                  $load_to_check = round($this->json['system_load'][2], 2);
                  break;
```

**Versions**
* openITCOKPIT Server Version: 4.6.8

**Comment by nook24** [comment]
This is fixed in the current nightly build - and also within the upcoming stable

**Comment by ibering** [comment]
This issue was resolved with version [4.6.10](https://github.com/it-novum/openITCOCKPIT/releases/tag/openITCOCKPIT-4.6.10) behoben
